Makefile

```diff
@@ -33,10 +34,14 @@ LDFLAGS += -X $(GO_PROJECT)/pkg/util.GitCommit=$(GIT_COMMIT)
 LDFLAGS += -X $(GO_PROJECT)/pkg/util.DriverVersion=$(CSI_IMAGE_VERSION)
 
 # set GOARCH explicitly for cross building, default to native architecture
-ifeq ($(origin GOARCH), undefined)
+ifndef GOARCH
 GOARCH := $(shell go env GOARCH)
 endif
 
+ifndef BASE_IMAGE
```

Comment:
It is probably better to do:
```
ifdef BASE_IMAGE
BASE_IMAGE_ARG = --build-arg BASE_IMAGE=$(BASE_IMAGE)
endif
```
and use `$(BASE_IMAGE_ARG)` at the `image-cephcsi` target below. Removes yet an other hardcoded version.